apps/sms/views/conversation/js/conversation.js

```diff
@@ -1916,9 +1916,15 @@ var ConversationView = {
     }
     params.items.push(subjectItem);
 
-    // If we are on a thread, we can call to SelectMessages
+    /*
+     * If we are on a thread,
```

Comment:
__nit:__ mm having ```//``` comment that occupies all available space - up to 80 chars - feels better